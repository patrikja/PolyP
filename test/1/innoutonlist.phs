outhej = out "hej"
main = print (outhej, inn outhej)
-- expected output: (Right ('h',"ej"),"hej")
