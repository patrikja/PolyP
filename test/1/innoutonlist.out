(InR (ParF{unParF='h'} :*: RecF{unRecF="ej"}),"hej")
